config/v3/configV3.go

```diff
@@ -211,6 +213,10 @@ func applyImportDefaults(metricConfig *MetricConfig, defaults DefaultConfig) {
 	if metricConfig.Type == "summary" && len(metricConfig.Quantiles) == 0 {
 		metricConfig.Quantiles = defaults.Quantiles
 	}
+	if metricConfig.Type == "summary" && metricConfig.MaxAge == 0 {
+		fmt.Print("set max age default")
```

Comment:
please remove :)

Reply:
Oh, sorry, didn't notice that one.
Removed now.